src/default-provider.ts

```diff
@@ -0,0 +1,10 @@
+import { Network, Networkish } from '@ethersproject/networks'
+import { BaseProvider, FallbackProvider } from '@ethersproject/providers'
+
+export function getDefaultProvider(network: Networkish | Promise<Network>): BaseProvider {
+  const defaultProviders: Array<BaseProvider> = []
+
+  defaultProviders.push(new BaseProvider(network))
```

Comment:
you should add a NearProvider here. No a BaseProvider

Reply:
Done!